code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -53,7 +53,7 @@
 	to_chat(owner, "<span class='userdanger'>You suddenly feel an irresistible compulsion to follow an order...</span>")
 	to_chat(owner, "<span class='mind_control'>[command]</span>")
 	active_mind_control = TRUE
-	log_admin("[key_name(user)] sent an abductor mind control message to [key_name(owner)]: [command]")
+	message_admins("[key_name_admin(user)] sent an abductor mind control message to [key_name_admin(owner)]: [command]")
```

Comment:
```suggestion
	log_admin("[key_name(user)] sent an abductor mind control message to [key_name(owner)]: [command]")
	message_admins("[key_name_admin(user)] sent an abductor mind control message to [key_name_admin(owner)]: [command]")
	user.create_log(CONVERSION_LOG, "sent an abductor mind control message: '[command]'", owner)
	owner.create_log(CONVERSION_LOG, "received an abductor mind control message: '[command]'", user)
```
Would be wise to still keep the log and also add in the logs which the ingame log viewer uses

Reply:
Sure